### DISCUSSION TITLE
Managing OpenSearch users using CRDs and securityconfig

### DISCUSSION THREAD

**Discussion by nilushancosta** [Opster/opensearch-k8s-operator#915] [Q&A] [answered]
The OpenSearch operator [Readme](https://github.com/opensearch-project/opensearch-k8s-operator/blob/main/docs/userguide/main.md#user-and-role-management) states the following
```
Note that currently a combination of both approaches is not possible. Once you use the CRDs you cannot provide your own securityconfig as those would overwrite each other. We are working on a feature to merge these options.
```

However when I tested by using a combination of both, things worked as expected. Could I not have encountered this overwriting stage yet or has this feature been implemented and a documentation update is required?

**Comment by swoehrl-mw** [comment, marked as answer]
Hi @nilushancosta. This problem still exists. You likely just have not encountered it as it only surfaces if you make a change to the securityconfig after configuring users via CRD.

**Reply by nilushancosta** [reply]
thanks for the response @swoehrl-mw . I didn't make a change to the securityconfig after configuring users via the CRD. Therefore I will switch to the YAML based user management approach for now
